cmake/Modules/FindNCCL.cmake

```diff
@@ -1,4 +1,4 @@
-# Find the nccl libraries
```

Comment:
For all these changes you'll have to recreate the docker images first

Reply:
Why exactly?

Reply:
Actually after looking further I don't think that this is the problem.